extension-manifest-v3/scripts/build.js

```diff
@@ -145,6 +145,13 @@ if (manifest.declarative_net_request?.rule_resources) {
   });
 }
 
+// copy TrackerDB
```

Comment:
This is not required, as whole `src/assets` path is copied in L73. I think it might even support subpaths. It takes a list of paths from L13.